habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -0,0 +1,519 @@
+import os
```

Comment:
This is also another file which is really heavy in terms of code size, I have made suggestions below to abstract out the logics to other modules.

Reply:
Discussed offline. Will make more abstraction to `ppo_trainer.py`in a separate PR, stuff like moving ppo update and rollout stage out of `train()`

Reply:
Agree with @abhiskk and @JasonJiazhiZhang need refactoring in next PR.